### ISSUE TITLE
Error: No game mode detected

### ISSUE THREAD

**Issue by aphex3k** [nicehash/NiceHashQuickMiner#245]
When trying to switch on Game Mode, I get the following error:
![Screenshot 2021-03-19 131736](https://user-images.githubusercontent.com/433270/111837743-924e5f80-88b5-11eb-9af6-71a419686f69.jpg)

```
[2021-03-19 13:16:56:159] [INFO] [main] Activating game mode
[2021-03-19 13:16:56:159] [TRACE] [watchdog] HTTP API GET: /api?command={"id":1,"method":"device.get","params":["0"]}
[2021-03-19 13:16:56:187] [DEBUG] [watchdog] Setting target: STOPPED
[2021-03-19 13:16:56:188] [INFO] [watchdog] New target: STOPPED
[2021-03-19 13:16:56:188] [TRACE] [watchdog] HTTP API GET: /setmaxcoreclock?id=0&core=0
[2021-03-19 13:16:56:195] [WARNING] [watchdog] Setting game state, error: Object reference not set to an instance of an object.
[2021-03-19 13:16:56:196] [WARNING] [main] Detected error: Setting game state error: Object reference not set to an instance of an object.
[2021-03-19 13:16:56:276] [TRACE] [nhmws] Received: {"method":"sma","data":[[20,"0.001713957481"],[52,"0.002373686817"],[24,"1.028325362"],[51,"1e-07"],[50,"100105.1721"],[21,"2.696105074e-09"],[7,"1e-07"],[46,"5.812096227e-05"],[8,"0.01517791858"],[23,"1.844144532e-06"],[48,"1.551970591e-07"],[43,"6519.230127"],[36,"443.1137172"],[5,"5.951959245e-07"],[28,"3.757508494e-08"],[47,"1.422387552"],[54,"1749.572645"],[33,"0.0001694638779"],[14,"1.918618821e-06"],[39,"21593.0303"],[42,"5.5"],[56,"0.001724993326"],[32,"0.00025025898"],[40,"0"]],"stable":"[20]"}
[2021-03-19 13:16:56:611] [INFO] [watchdog] Starting/stopping with: /quickstop
[2021-03-19 13:16:56:611] [TRACE] [watchdog] HTTP API GET: /quickstop
[2021-03-19 13:16:56:612] [TRACE] [watchdog] HTTP API GET: /disablecpu
[2021-03-19 13:17:03:544] [INFO] [main] Deactivating game mode
[2021-03-19 13:17:03:544] [WARNING] [main] Detected error: No game mode detected
```

- v0.4.1.3 (latest)
- [exported_logs.zip](https://github.com/nicehash/NiceHashQuickMiner/files/6173832/exported_logs.zip)

**Comment by aphex3k** [comment]
This issue seems resolved in `v0.4.2.0 RC`

**Comment by nicehashdev** [comment]
Resolved, fixed.
